example/LLM/bloom/download_model.py

```diff
@@ -18,5 +18,8 @@ def download():
     del tokenizer
 
 
-if __name__ == "main":
-    download()
+if __name__ == "__main__":
+    if sys.argv[1]:
```

Comment:
if we don't provide argument, `sys.argv[1]` will raise `IndexError` exception.

```
if len(sys.argv) == 2:
  download(sys.argv[1])
else:
  download()
```
?